webapps/ui/cockpit/plugins/base/app/views/processDefinition/diagramPlugins/instanceCount.js

```diff
@@ -88,11 +89,11 @@ module.exports = [
             }
 
             function removeOverlays() {
-              Array.from(
-                document.getElementsByClassName('djs-overlay')
```

Comment:
This would also remove all other overlays on the diagram, so we have to remember the overlays we actually want to remove.